Jenkinsfile

```diff
@@ -81,12 +81,16 @@ node('psi_rhel8') {
             docker.image('circleci/android:api-28-node').inside('-u root:root --network aerogear') {
               dir('aerogear-integration-tests') {
                 sh 'npm set registry http://verdaccio:4873/'
-                sh 'apt install gradle'
                 sh 'npm -g install cordova@8'
                 checkout scm
                 withCredentials([file(credentialsId: 'google-services', variable: 'GOOGLE_SERVICES')]) {
                   sh 'cp ${GOOGLE_SERVICES} ./fixtures/google-services.json'
-                  sh './scripts/build-testing-app.sh'
+                  sh '''#!/bin/bash -l
```

Comment:
I think is better to just do `apt update` and `apt install gradle`, probably they have clean the apt cache with the latest version of the circle ci container

Reply:
Didn't know that this could help, makes sense, will change it.

Reply:
I tried on my machine and it works `docker run -it circleci/android:api-28-node bash` and then `sudo apt update` and `sudo apt install gradle`